roboto/http_api.py

```diff
@@ -66,6 +69,53 @@ def validate_response(response: APIResponse) -> Any:
     return response.result
 
 
+async def _json_request(
+    session: Session, method: HTTPMethod, api_method: str, body: Any = None
+) -> Response:
+    return await session.request(method.value, path=api_method, json=to_json(body))
+
+
+def _multipart_compatible(value: Any) -> Any:
```

Comment:
It took me some time to understand that this function is making value to be multipart_compatible, maybe rename it to `_to_multipart_compatible` or `_makes_multipart_compatible`

Reply:
Reasonable.